index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
-          providers for the most common managed services.
+          Securely consume managed services such as PostgreSQL, Redis, Buckets and more, from 
+          your choice of cloud on-premises.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
         </p>
-        <p><img src="{{ "/images/pink.svg" | relative_url }}" /></p>
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/compute.svg" | relative_url }}" /></div>
-        <h4>Compute</h4>
+    </div>
+    <div class="grid">
+      <div class="col-6_md-12">
+        <h2>Extend Crossplane by adding your own Stack.</h2>
+        <p>
+          Add new resources and controllers to Crossplane by extending existing Stacks
+          or building your own.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
+        </p>
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/database.svg" | relative_url }}" /></div>
-        <h4>Databases</h4>
+      <div class="col-6_md-12_md-first img">
+        <img srcset="./images/feature-4.png 1x, ./images/feature-4@2x.png 2x, ./images/feature-4@3x.png 3x"
+          alt="Extend Crossplane by adding your own Stack" />
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/storage.svg" | relative_url }}" /></div>
-        <h4>Storage</h4>
+    </div>
+    <div class="grid">
+      <div class="col-6_md-12 img">
+        <img srcset="./images/feature-2.png 1x, ./images/feature-2@2x.png 2x, ./images/feature-2@3x.png 3x"
+          alt="Schedule workloads across clusters, regions and clouds" />
       </div>
-      <div class="col-9_md-12 footer">
+      <div class="col-6_md-12">
+        <h2>Schedule workloads across clusters, regions and clouds.</h2>
         <p>
-          Additional managed services including serverless, analytics, big data,
-          machine learning and others are coming soon.
+          Define complete applications with managed service dependencies for scheduling across 
+          clusters and regions to balance reliability, cost, and performance.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
         </p>
+      </div>
+    </div>
+    <div class="grid">
+      <div class="col-6_md-12">
+        <h2>Manage multiple clusters from a single control plane.</h2>
         <p>
-          <a href="{{githubLink}}"
-            >Help support these features and add new ones</a
-          >
+          Provision and manage kubernetes clusters: bring your own clusters or generate on demand, 
+          with common configuration and policy.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
         </p>
       </div>
+      <div class="col-6_md-12_md-first img">
+        <img srcset="./images/feature-3.png 1x, ./images/feature-3@2x.png 2x, ./images/feature-3@3x.png 3x"
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper